### PR TITLE
feat: FullStory

### PR DESCRIPTION
**Pull request by eladb**
Record user sessions with FullStory.

### CODE REVIEW

File: src/analytics.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+import * as amplitude from "@amplitude/analytics-browser";
+import * as FullStory from '@fullstory/browser';
 
-export async function setupAnalytics(amplitudeApiKey: string) {
-  await init(amplitudeApiKey, undefined, {
+const AMPLITUDE_API_KEY = "0dd9d988f0137cbb9d3bd337a00fe257";
+const FULLSTORY_ORG_ID = "o-1CTP1V-na1";
+
+export async function setupAnalytics() {
+  FullStory.init({ orgId: FULLSTORY_ORG_ID });
```
Comment (skyrpex) [inline src/analytics.ts:8]: Does `FullStory` use cookies? We don't have a cookies modal right now.

Comment (eladb) [reply src/analytics.ts:8]: It uses a [first-party cookie]


[first-party cookie]: https://help.fullstory.com/hc/en-us/articles/360020829513-Why-FullStory-uses-First-Party-Cookies

I guess we need to add cookie consent, ha?

Comment (eladb) [reply src/analytics.ts:8]: https://github.com/monadahq/website/issues/6

**Comment by eladb** [comment]
Changing to draft until #6 is resolved.

**Comment by eladb** [comment]
Superseded by #7

**Pull request closed without merging**